File: project_euler/problem010.py
# The sum of the primes below 10 is 2 + 3 + 5 + 7 = 17.
#
# Find the sum of all the primes below two million.

from utils import number_theory

def problem010():
    return sum(number_theory.primes_up_to(2_000_000))

File: typescript/src/problems/problem_010.ts
import * as utils from "../utils";

// Problem 10:
// The sum of the primes below 10 is 2 + 3 + 5 + 7 = 17.
// Find the sum of all the primes below two million.

export function main(): number {
    return utils
        .primesUpTo(2_000_000)
        .reduce((total, i) => total += i, 0)
}
